wallet-new/src/Cardano/Wallet/Kernel/Pending.hs

```diff
@@ -92,19 +92,22 @@ newTx :: forall e. ActiveWallet
       -> IO (Either e TxMeta)
 newTx ActiveWallet{..} accountId tx partialMeta upd = do
     -- run the update
-    allOurs' <- allOurs <$> getWalletCredentials walletPassive
-    let (addrsOurs',ourOutputCoins) = unzip allOurs'
-        gainedOutputCoins = sumCoinsUnsafe ourOutputCoins
-        allOutsOurs = length allOurs' == length txOut
+    allCredentials <- getWalletCredentials walletPassive
+    let addrsOurs' = fst <$> allOurs allCredentials
```

Comment:
Personal preference on style: wouldn't be better to name this function `allOurAddresses` ? I'm personally not a fan of the `foo / foo'` pattern. I think it has use only in 2 circumstances:

- You want to differentiate between a lazy vs strict variant of a function (cfr. `foldl` vs `foldl'`)
- You want to stress the fact something named equally has been modified by a function (example `snapshot` and `snapshot'`).

As a `'` is very easy to miss and forget, I tend to use them sparingly 😉